### ISSUE TITLE
ALT+ENTER cheat doesn't work

### ISSUE THREAD

**Issue by JAsmine-ja2** [ja2-stracciatella/ja2-stracciatella#504]
ALT+ENTER cheat (end enemie's turn) doesn't work, because this key combination switches between fullscreen and windowed mode in Stracciatella.

Stracciatella 0.15.1, Windows 64-bit

**Comment by Manuel-K** [comment]
It could be changed to CTRL+ENTER…

**Comment by JAsmine-ja2** [comment]
True. Another idea would be ALT+D, because D is the vanilla hotkey for ending the player's turn (D = "Done").

**Comment by Manuel-K** [comment]
This seems to be the the code in question: https://github.com/ja2-stracciatella/ja2-stracciatella/blob/master/src/game/Tactical/Turn_Based_Input.cc#L2101

CTRL+ENTER is used to call `EscapeUILock()`.
ALT+D is already in use, too. It is defined here: https://github.com/ja2-stracciatella/ja2-stracciatella/blob/master/src/game/Tactical/Turn_Based_Input.cc#L1357

**Comment by icelynjennings** [comment]
For maximum vanilla recreation, perhaps the Stracciatella fullscreen hotkey should be changed, not the original 1.12 cheat combination?

**Comment by lynxlynxlynx** [comment]
By now there are more people familiar with the fullscreen key than the rare few that use this cheat, so it would be a different regression of sorts.

**Comment by momoko-h** [comment]
Just for reference if someone's looking for this cheat: the new key combination (after activating cheats) is ALT + 6.
